inc/namespace.php

```diff
@@ -229,3 +232,12 @@ function robots_txt( string $output ) : string {
 
 	return $output;
 }
+
+/**
+ * Enqueue CSS.
+ */
+function enqueue_yoast_css_overrides() {
+	$version = ( defined( 'WP_DEBUG' ) && WP_DEBUG ) || ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ) ? Altis\get_version() . '?' . time( 'Ymd' ) : Altis\get_version();
```

Comment:
This makes me think that the `Altis\get_version()` version should support a way to get a specific module's version too.

I don't usually see the pattern of using `WP_DEBUG` or `SCRIPT_DEBUG`, usually just `SCRIPT_DEBUG` is appropriate for this kind of logic.

In most places we've just hardcoded a date string for the version in the format `Y-m-d-<version>` where `<version>` starts at 1 and increments if you need to commit more than one modification in a single day.

The reason for using the above format is that this stylesheet will not be likely to change that often, and isn't strictly bound to the current Altis version so there's not really anything to gain by making it dynamic. If you have the inspector / dev tools open you can disable the cache (if not disabled by default) rather than relying on code for a time based version string.

Lastly `time()` doesn't accept any parameters, it'll always just return the current timestamp.

Reply:
There was a difference in behavior that I was noticing with some Yoast modules (particularly when trying to test a "production"-ish environment) between `SCRIPT_DEBUG` and `WP_DEBUG`, which is why I was checking for both, but I think it's probably no longer necessary outside of development.